rgw/v2/tests/curl/configs/test_cors_using_curl.yaml

```diff
@@ -0,0 +1,25 @@
+# script: test_cors_using_curl.py
+# polarion: CEPH-10355
+config:
```

Comment:
description has polarion id : CEPH-8357557

Reply:
@anrao19 the description was wrong , updated it, thanks